src/java.desktop/unix/classes/sun/awt/X11/XCheckboxMenuItemPeer.java

```diff
@@ -49,7 +49,7 @@ final class XCheckboxMenuItemPeer extends XMenuItemPeer
      *
      ************************************************/
 
-    //Prom CheckboxMenuItemtPeer
+    //Prom CheckboxMenuItemPeer
```

Comment:
```suggestion
    //From CheckboxMenuItemPeer
```
However, I'd rather remove the comment altogether: the IDE shows that it implements an interface, `CheckboxMenuItemPeer` in this case.